README.md

```diff
@@ -411,6 +426,14 @@ function handleRoute (req, res, ctx, done) {
 }
 ```
 
+### app.use(plugin)
+Register a plugin on the merry instance. A plugin is an object where the
+following values are possible:
```

Comment:
do you also think we should have an example here as well, or just under `plugins`?